### DISCUSSION TITLE
Can this be your default text editor In Windows 10?

### DISCUSSION THREAD

**Discussion by Eastdoes** [atom/atom#22661] [Q&A]
If so please tell me how.

**Comment by icecream17** [comment]
Ah google and stackoverflow!
![image](https://user-images.githubusercontent.com/58114641/123526164-9b608100-d69b-11eb-85f4-c40e2d2e2850.png)

https://stackoverflow.com/questions/27617208/how-do-i-make-atom-my-default-editor
